README.md

```diff
@@ -59,36 +59,22 @@ make docker run_docker
 ```
 
 With a modern browser, navigate to http://yourdomain:4000 (change *yourdomain* to your server's hostname or ip). 
-> NOTE: make sure that EdgeXFoundry is already running, Please refer to [Edgex Services Get Started](https://github.com/edgexfoundry/edgex-go/blob/master/README.md). 
-
-> NOTE: if you want remote access the UI, you shuold remove the UI network in docker-compose.yml, this way you can allow the UI to live someplace else and have access to EdgeX through Kong.
-
-If you want to run edgex-ui-go with docker-compose yml file, you can reference three ways:
-
-> 1: For example [docker-compose-hanoi-ui](https://github.com/edgexfoundry/developer-scripts/blob/master/releases/hanoi/compose-files/docker-compose-hanoi-ui.yml), you can change 'networks' to the same as [docker-compose-hanoi](https://github.com/edgexfoundry/developer-scripts/blob/master/releases/hanoi/compose-files/docker-compose-hanoi.yml), and run 'docker-compose -f docker-compose-hanoi-ui.yml up -d', then the UI will connect with EdgeX Server.
- 
-> 2: You can use the make file commands ‘make run arm64’ and ‘make run-ui arm64’ to run the Edgex Stack and UI as it uses the ‘-p edgex’ option required for the networking to link properly. 
-
-> 3: If not running from the cloned developer-scripts, you can also do this directly with ‘docker-compose -p edgex -f docker-compose-hanoi-arm64.yml up -d’ and ‘docker-compose -p edgex -f docker-compose-hanoi-ui-arm64.yml up -d’.
+> NOTE: make sure that EdgeXFoundry is already running, Please refer to [Edgex Services Get Started](https://github.com/edgexfoundry/edgex-go/blob/main/README.md). 
 
 ### Snap 
 
 EdgeX UI is also available as a snap package. For more details on the snap, including how to install it, please refer to [EdgeX UI Snap](https://github.com/edgexfoundry/edgex-ui-go/tree/main/snap/README.md)
 
 ## Community
 - EdgeXFoundry Chinese Club: https://www.edgexfoundry.club
-- Chat: https://chat.edgexfoundry.org/home
-- Mainling lists: https://lists.edgexfoundry.org/mailman/listinfo
+- Chat: https://edgex.slack.com
```

Comment:
Please add join link: https://slack.edgexfoundry.org/

Reply:
what's  different from this link https://edgex.slack.com

Reply:
slack.edgexfoundry.org issues you and invite to join the slack project.

edgex.slack.com allows you to log in to slack from the web ui once you are already joined.

Reply:
ok, thanks, I will update it

Reply:
@bnevis-i it's done, please review it again